skimage/segmentation/tests/test_join.py

```diff
@@ -114,7 +114,7 @@ def test_relabel_sequential_signed_overflow():
 
 
 def test_very_large_labels():
-    imax = np.iinfo(np.int64).max
+    imax = np.iinfo(np.int32).max
     labels = np.array([0, 1, imax, 42, 42], dtype=np.int64)
     output, fw, inv = relabel_sequential(labels, offset=imax)
     assert np.max(output) == imax + 2
```

Comment:
Maybe I'm misunderstanding what was intended by this test, but if we specify `imax = np.iinfo(np.int64).max` and then add an offset to the new labels of `offset=imax`, numpy can't accurately distinguish between `imax + x` where `x > 0`. I've changed the test to use a "smaller" large number so the `np.int64` doesn't overflow immediately within `relabel_sequential`.

Reply:
This one needed to stay `np.int64`. I updated the logic within `relabel_sequential` in https://github.com/scikit-image/scikit-image/pull/6637/commits/bb7238c8f358041738c30c68735232670912bf1e of #6637